From: LineageOS Builder <builder@lineageos.local>
Date: Sun, 7 Jun 2026 20:30:00 +0800
Subject: [PATCH] Allow SELINUX_IGNORE_NEVERALLOWS in user builds for GSI

diff --git a/Android.mk b/Android.mk
index dadd7b0a..197f25c5 100644
--- a/Android.mk
+++ b/Android.mk
@@ -83,12 +83,7 @@ endif
 
 NEVERALLOW_ARG :=
 ifeq ($(SELINUX_IGNORE_NEVERALLOWS),true)
-ifeq ($(TARGET_BUILD_VARIANT),user)
-$(error SELINUX_IGNORE_NEVERALLOWS := true cannot be used in user builds)
-endif
-$(warning Be careful when using the SELINUX_IGNORE_NEVERALLOWS flag. \
-          It does not work in user builds and using it will \
-          not stop you from failing CTS.)
+# Allow user builds too for GSI (removed user restriction)
 NEVERALLOW_ARG := -N
 endif
-- 
2.25.1
